src/NodaTime/Text/Patterns/SteppedPatternBuilder.cs

```diff
@@ -247,13 +247,15 @@ internal static void HandleBackslash(PatternCursor pattern, SteppedPatternBuilde
         }
 
 #pragma warning disable IDE0060
```

Comment:
This appears to be the same as CA1801. Do we need both? (Do we need this elsewhere?)